lib/elasticity/strategies/alias_index.rb

```diff
@@ -238,6 +238,14 @@ def settings
       end
 
       def mappings
+        Elasticity::LogSubscriber.logger.warn(
+          'Elasticity::Strategies::AliasIndex#mappings is deprecated, '\
+          'use mapping instead'
+        )
+        mapping
+      end
```

Comment:
`LogSubscriber` is not supposed to be called directly. The whole idea with `ActiveSupport::LogSubscriber` is that your code doesn't call it directly, it logs things based on the instrumentation. See http://guides.rubyonrails.org/active_support_instrumentation.html

For this case specifically just use `deprecate :mappings`, if you wanna customize the output you can use a custom deprecator. See http://api.rubyonrails.org/classes/Module.html#method-i-deprecate